.github/workflows/tests.yaml

```diff
@@ -0,0 +1,52 @@
+name: sdk-tests
+
+on:
+  workflow_dispatch:
+    inputs:
+      network:
+        description: 'MobileCoin Network'
+        required: true
+        default: 'MOBILE_DEV'
+        type: choice
+        options:
+          - 'MOBILE_DEV'
+          - 'ALPHA'
+          - 'TEST_NET'
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: GCP Service Account
+        run: |
+          echo '${{ secrets.GCP_SERVICE_KEY }}' > "service-key.json"
+
+      - name: Provide Dev Credentials
+        env:
+          TEST_DEV_USER: ${{ secrets.TEST_DEV_USER }}
+          TEST_DEV_PASSWORD: ${{ secrets.TEST_DEV_PASSWORD }}
+        run: |
+          sed -i 's/REPLACE_TEST_DEV_USER_STRING/'"${TEST_DEV_USER}"'/g' android-sdk/src/androidTest/java/com/mobilecoin/lib/TestFogConfig.java && \
+          sed -i 's/REPLACE_TEST_DEV_PASSWORD_STRING/'"${TEST_DEV_PASSWORD}"'/g' android-sdk/src/androidTest/java/com/mobilecoin/lib/TestFogConfig.java
+
+      - name: Set the Network
+        env:
+          NETWORK_UNDER_TEST: ${{ github.event.inputs.network }}
+        run: sed -i 's/TestEnvironment.ALPHA/'"TestEnvironment.${NETWORK_UNDER_TEST}"'/g' android-sdk/src/androidTest/java/com/mobilecoin/lib/Environment.java
+
+      - name: Provide DevNet Entropies
```

Comment:
This seems odd that there are 2 different ways to specify the wallet/key entropy

Reply:
All networks used to have root-entropy based accounts, but it didn't support 24 words recovery, so we've upgraded to slip10. Dev nets haven't been updated yet, that's why two ways of supplying accounts.